test/Dockerfile

```diff
@@ -1,11 +1,5 @@
-FROM bash:5.0.17
+FROM powerman/dockerize
 
-RUN apk add --no-cache openssl curl
+RUN apk add --no-cache openssl curl bash
 
-ENV DOCKERIZE_VERSION v0.6.1
-RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
```

Comment:
[Dockerize](https://github.com/jwilder/dockerize) hasn't had any commits/releases since end of 2021 (which is generally fine for test code) but it doesn't support multi-platform which is a bit more a problem for me and this PR.


Have switched to 

- https://github.com/powerman/dockerize  
- https://hub.docker.com/r/powerman/dockerize/

which looks to be a decent replacement